gap/SnAnUnknownDegree.gi

```diff
@@ -0,0 +1,295 @@
+# Input: Group G, upper error bound eps, upper degree bound N
+#
+# The following algorithm constructs a set of possible 3-cycles. It is based
+# on the simple observation that the product of two involutions t1, t2, which
+# only move one common point, squares to a 3-cycle.
+#
+# Either returns a list of elements of G or NeverApplicable
+BindGlobal("ThreeCycleCandidates",
+function(G, eps, N, groupIsOne, groupIsEq)
```

Comment:
We will change this at the end. For now we leave it as it is so we can test the functions more easily in the console.

Reply:
Actually, we'll try to do this next. Sounds good. :)

Reply:
If it is difficult to test it via the console when using a recog info record, then we should make it easier to test with a proper recog info record (e.g. by adding helpers to produce them for you). You might in fact want to think about creating such helpers. Because everybody developing a recog method needs to eventually use those, and the sooner, the better (you avoid the time to rewrite the code from one API to another, and the risk of subtle mismatches). Also, all unit tests need that.